src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator-container">
+      <div className="calculator">
+        <input className="output-display" htmlFor="textarea" placeholder="0" disabled />
+        <div className="buttons-div">
+          <button className="buttons" type="button">AC</button>
+          <button className="buttons" type="button">+/-</button>
+          <button className="buttons" type="button">%</button>
```

Comment:
- [ ] Please make your `Calculator `component class-based as stated in the project requirements 👍